ndk/src/looper.rs

```diff
@@ -264,27 +265,35 @@ impl ForeignLooper {
     ///
     /// See also [the NDK
     /// docs](https://developer.android.com/ndk/reference/group/looper.html#alooper_addfd).
-    pub fn add_fd_with_callback(
+    pub fn add_fd_with_callback<F: FnMut(RawFd) -> bool>(
         &self,
         fd: RawFd,
         ident: i32,
         events: FdEvent,
-        callback: Box<dyn FnMut(RawFd) -> bool>,
+        callback: Box<F>,
```

Comment:
Whoops, I just realized that we can probably hide the `Box<F>` implementation detail (just accept `: F` here, and `Box::new()` that internally), and I should also add a changelog entry for this _breaking_ change. @msiglreith thoughts?

Reply:
right, that's better!